test/it/analysisTest/simpleProcessingChainTest.js

```diff
@@ -8,7 +8,7 @@ import * as SurveyManager from '@server/modules/survey/manager/surveyManager'
 import { getContextUser } from '../../testContext'
 import * as SB from '../utils/surveyBuilder'
 import * as RB from '../utils/recordBuilder'
-import * as ChainB from '../utils/processingChainBuilder'
+import * as ChainB from '../utils/chainBuilder'
```

Comment:
what i meant is the import * as ChainBuilder... i don't understand a variable called ChainB. also the class itself is good chainBuilder

Reply:
aaa ok, it was only to make the code below shorter, avoiding to repeat ChainBuilder.chain, ChainBuilder.step etc. every time... like we did for SB and RB